serverless/serverless
# Change version to function-version in rollback command

## What did you implement

Changed `version` option in rollback command to `function-version` to avoid confusion with `version` option from core

Closes #7645

## How can we verify it
```
service: test-rollback

provider:
    name: aws
    region: us-east-2
    runtime: nodejs12.x
    stage: ${env:NODE_ENV}
    timeout: 900
functions:
  app:
    handler: index.handler
```

If you try to rollback with the following command before the fix, it will throw an error that `1 is not a sub command of version`
`npx serverless rollback -f app --version 1`

If you try this command after the fix, it will not throw that error
`npx serverless rollback -f app --function-version 1`

_Please note that this test keeps failing on my side (I'm using windows currently) `calls docker with packaged artifact`_

## Todos

- I'm not sure if we should change/remove the shortcut `v`

<details>
<summary>Useful Scripts</summary>


- `npm run test:ci` --> Run all validation checks on proposed changes
- `npm run lint:updated` --> Lint all the updated files
- `npm run lint:fix` --> Automatically fix lint problems (if possible)
- `npm run prettier-check:updated` --> Check if updated files adhere to Prettier config
- `npm run prettify:updated` --> Prettify all the updated files

</details>


**_Is this ready for review?:_** NO
**_Is it a breaking change?:_** YES
